dwave/embedding/chain_strengths.py

```diff
@@ -47,3 +63,45 @@ def uniform_torque_compensation(bqm, embedding=None, prefactor=1.414):
         return prefactor * rms * math.sqrt(avg_degree)
     else:
         return 1    # won't matter (chain strength isn't needed to embed this problem)
+
+def scaled_to_problem(bqm, embedding=None, prefactor=1.0):
+    """Chain strength that is scaled to the problem bias range.
+
+    Args:
+        bqm (:obj:`.BinaryQuadraticModel`):
+            A binary quadratic model.
+
+        embedding (dict/:class:`.EmbeddedStructure`, default=None):
+            Included to satisfy the `chain_strength` callable specifications 
+            for `embed_bqm`. 
+
+        prefactor (float, optional, default=1.0):
+            Prefactor used for scaling. 
+
+    Returns:
+        float: The chain strength, or 1 if chain strength is not applicable
+               to the problem. 
+
+    Examples:
+        This example uses :func:`scaled_to_problem`, given a prefactor of 1.5, 
+        to calculate a chain strength that :class:`EmbeddingComposite` then uses.
+
+        >>> from functools import partial
+        >>> from dwave.system import EmbeddingComposite, DWaveSampler
+        >>> from dwave.embedding.chain_strengths import scaled_to_problem
+        ...
+        >>> Q = {(0,0): 1, (1,1): 1, (2,3): 2, (1,2): -2, (0,3): -2}
+        >>> sampler = EmbeddingComposite(DWaveSampler())
+        >>> # partial() can be used when the BQM or embedding is not accessible
+        >>> chain_strength = partial(scaled_to_problem, prefactor=1.5)
+        >>> sampleset = sampler.sample_qubo(Q, chain_strength=chain_strength, return_embedding=True)
+        >>> sampleset.info['embedding_context']['chain_strength']
+        0.75
+
+    """  
+    if bqm.num_interactions > 0:
+        max_bias = max(max(map(abs, bqm.linear.values()), default=0),
+                       max(map(abs, bqm.quadratic.values()), default=0))
```

Comment:
Since we already have (a very fast) linear/quadratic max/min, this should be _much_ faster:
```
maxabs = lambda *items: max(map(abs, items), default=0)
max_bias = maxabs(bqm.linear.max(), bqm.linear.min(),
                  bqm.quadratic.max(), bqm.quadratic.min())
```